mailpoet/.babelrc

```diff
@@ -1,8 +1,8 @@
 {
   "presets": [
-    "@babel/preset-typescript",
     ["@babel/preset-react", { "runtime": "automatic" }],
-    "@babel/preset-env"
+    "@babel/preset-env",
+    ["@babel/preset-typescript", { "allowDeclareFields": true }]
```

Comment:
I wonder whether we should review the preset order in any other config (for any reason)...